src/DynamoCore/PackageManager/PackageUploadBuilder.cs

```diff
@@ -1,205 +1,93 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
 using System.Reflection;
-using Dynamo.Core;
-using Dynamo.Models;
-using Dynamo.Utilities;
+using System.Text;
 using Greg.Requests;
-using RestSharp.Serializers;
 
 namespace Dynamo.PackageManager
 {
-    static class PackageUploadBuilder
+    internal class PackageUploadBuilder
     {
```

Comment:
This is the most substantially changed file in this PR.  Most of it's guts are moved to `PackageDirectoryBuilder`.